src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/BaseAnalysisController.java

```diff
@@ -479,9 +483,9 @@ public JsonResult updateCommonEntityInAnalysis(@PathVariable("analysisId") Long
         if (order == null) {
             order = Sort.Direction.ASC;
         }
-        final SubmissionInsight insight = analysisService.getSubmissionInsight(submissionId);
+        final SubmissionInsight insight = submissionInsightService.getSubmissionInsight(submissionId);
```

Comment:
I think much more can be grabbed with this entity: user, files, result files, datasource, submission, analysis, submissionInsightSubmissionFiles, or it will take quite dohera queries.